src/main/java/io/github/mountainrange/mule/enums/MapType.java

```diff
@@ -4,5 +4,40 @@
  * List of map types (for the map generator).
  */
 public enum MapType {
-	CLASSIC, RANDOM, EXPERIMENTAL
+	CLASSIC("classic"),
+	RANDOM("random"),
+	EXPERIMENTAL("experimental"),
+	EMPTY("empty");
+
+	public final TerrainType[][] map;
+
+
+	MapType(String name) {
+		// Abbreviations
+		final TerrainType T99 = null;
+		final TerrainType T00 = TerrainType.PLAIN;
+		final TerrainType T01 = TerrainType.RIVER;
+		final TerrainType T02 = TerrainType.MOUNTAIN1;
+		final TerrainType T03 = TerrainType.MOUNTAIN2;
+		final TerrainType T04 = TerrainType.MOUNTAIN3;
+		final TerrainType T05 = TerrainType.TOWN;
+
+		if (name.equalsIgnoreCase("empty")) {
```

Comment:
I think you *might* be able to use `toString` and avoid having to define the `String name` parameter.